lib/resources/content/tsconfig.json

```diff
@@ -10,7 +10,13 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "moduleResolution": "node",
-    "lib": ["es2017", "dom"]
+    "lib": ["es2017", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "./src/*"
```

Comment:
I just took a look with a new babel project: It appears as though this does not make things inconsistent but rather makes the TS behavior the same as the ES one: import {App} from 'app'; in a fresh cli babel setup works as expected.